capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,90 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/** UniversityData is a class representing a single university. */
+@AutoValue
+public abstract class UniversityData {
+
+  /**
+   * Creates a new UniversityData
+   *
+   * @param name The human-readable name for the school/college/university.
+   * @param city The human-readable name for the city the university is located in.
+   * @param latitude A float representing the school's location measured in latitude.
+   * @param longitude A float representing the school's location measured in longitude.
+   * @param flagMainCampus An integer that flags whether the university is the main campus or not, 1
+   *     or 0.
+   * @param urbanizationDegree An integer from range(1-8) that tells the degree of urbanization for
+   *     that campus, from large city to rural.
+   * @param admissionRate The overall rate of admission for the university.
+   * @param avgSat The average overall SAT score for students accepted to the university.
+   * @param numOfUndergrads The number of undergraduate students that attend the university.
+   * @param avgCost The average cost of attendance for students at the university.
+   * @param studentsInDebt The number of students in cumulative loan debt after graduation.
+   * @param carnegieSizeDegree An integer from range(-2-18) that tells degree of Carnegie size, from
+   *     not applicable to exclusively graduate/professional.
+   * @param numOfMen A float that represents the number of men enrolled at the undergraduate level.
+   * @param numOfWomen A float that represents the number of women enrolled at the undergradute
+   *     level.
+   */
+  public static UniversityData create(
```

Comment:
I imagine we won't ever need this factory method. Because we always want to build this class based on the scorecard api. So gson would be responsible for building this class and not this factory method. Here is [an example](https://sites.google.com/site/gson/gson-user-guide#TOC-Object-Examples).

@ccondit is `AutoValue` going to work with `gson` correctly? I think it needs a class with argument-less constructor.

Reply:
`AutoValue` should work with `gson`. I set this up in #33 for reading json config data from json into an autovalue.